Cargo.toml

```diff
@@ -22,7 +22,6 @@ members = [
     "opentelemetry-user-events-logs",
     "opentelemetry-user-events-metrics",
     "opentelemetry-zipkin",
-    "opentelemetry-zpages",
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/issues/1208 opened an issue to track the full removal. It was generally agreed in the SIG call to disable it in this PR to unblock this PR and get this merged.

Reply:
We were attempting to unlink this PR from being tied to this whole discussion about zpages by someone only disabling build. 
So that it wouldn't block this PR. 
I guess let's discuss the fate in #1208